western/description.md

```diff
@@ -1,29 +1,70 @@
+
+In 1603 the lawyer and hobby astronomer Johann Bayer from Augsburg, Germany, made a new celestial atlas, called Uranometria [#2]. In this atlas, he introduced a systematic naming of stars starting with alpha, the brightest, beta, the second brightest, gamma, the third ... within a consellation. Subsequently, it was established to name a star with a Greek letter and the Latin constellation name in casus genitive. 
+
+In the 18th century, the century of great atlases (Flamsteed, Fortin, Bode: [#3][#4]), new constellations were added and filled the gaps between northern constellations. Additionally, there were suggestions to create new constellations in the southern part of the sky which had not been visible to Ptolemy.  European astronomers visited the the European colonies and instead of asking the native people for their constellations, the Europeans were creative. This way, many new technical inventions of the early modern period were depicted in constellations (e.g. a chemical machine, a balloon, an air pump) as well as exotic animals (e.g. a chamaeleon, a tucan, a paradise bird). 
+
+The problem was that sometimes astronomers did not agree with each other and, thus, atlases showed deviations from each other.   
+
+After many centuries of astronomers taking liberties in interpretation of the Ptolemaic original, there were many variants of constellations. As constellations are a positioning system, i.e. astronomers describe positions in the sky relative to stars and stars relative to constellations (like "Betelgeuse", Orion's right shoulder), it is necessary to use constellation names and star names uniformly all over the world. 
+
+In the "Fin de siècle", there were upcoming suggestions in the IAU to unify the system of constellations and star names [#5].  
+
+As the IAU in 1928 only defined the boundaries of these areas [#4] and not the way of drawing artwork within them, several sub-cultures of painting figures and drawing simplified stickfigures were developed during the 20th and 21st century. We provide some of them as further sky cultures. 
+
 ### Constellations
 
-The Western culture divides the celestial sphere into 88 areas of various sizes called _constellations_, each with precise boundary, issued by the International Astronomical Union. These constellations have become the standard way to describe the sky, replacing similar sets in other sky cultures exhaustively in daily usage.
+This artificial sky culture is based mainly on the Ptolemaic tradition which had been used in all three Abrahamitic religions and, thus, was common in the Near East, in Europe and its colonies on all continents. In 1922, the IAU officially accepted the list of 88 constellation names and their official abbreviations with three letters. The Belgian priest E. Delporte got the task to define constellation boundaries according to coordinates. In 1928, he finished the work and the IAU accepted his delimitations [#1]. 
```

Comment:
hm... well, I meant "artificial" because a) the IAU GA defined a work package and someone who shall do it, and b) more important: this "someone" (E.Delporte) used primarily the ancient Greek const. but filled the gaps with constellations selected from a variety of suggestions from the centuries in between.